refs/schemas/priv/expected-behavior.md

```diff
@@ -0,0 +1,655 @@
+     - `TRANSPARENCY.PURPOSE` Demands: recommend status = `GRANTED`, and data = list of Purposes that are included in any of the [Privacy Scope Triples](#privacy-scope-triples) included in the **Restriction Scope**
+     - `REVOKE-CONSENT` Demands:
+         - If restricted to concrete consent IDs with a [Consent Restriction](./RFC-PRIV.md#consent-restriction), recommend status = `GRANTED` and recalculate Eligible Privacy Scope to drop any [Privacy Scope Triples](#privacy-scope-triples) that have been included as a result of Consents being revoked.
+         - If Demand is restricted by a Privacy Scope, recommend status = `GRANTED` and [update consents](#updateConsent)
+         - If no Restriction is given in the Demand, revoke all consents given by this Data Subject
+         - If only a Date Range restriction is present, recommend status = `GRANTED` and revoke all consents that have been collected in the given Date Range
+         - In other combinations of Restrictions, take the resulting **Restriction Scope**, recommend status = `GRANTED` and [update consents](#updateConsent) as if an `OBJECT` Demand has been made with the **Restriction Scope**
+
+     - `OBJECT`, `RESTRICT` Demands:
+         - Recommend status = `GRANTED`, and then, upon granting [update consents](#updateConsent), [recalculate Eligible Privacy Scope](#updateScope) by excluding **Restriction Scope** or limiting to the **Restriction Scope**
+         - Recommend deletion of the **Concerned Fragments**.
+
+     - `DELETE` Demands:
+         - If the **Concerned Fragments** set is empty, recommend status = `DENIED`, motive = `NO-SUCH-DATA`
+         - If it is non-empty:
+             - recommend for deletion the Data Capture Fragments from that set, the `scope` of which only includes [Privacy Scope Triples](#privacy-scope-triples) that are introduced in the Eligible Privacy Scope under Legal Bases belonging to {`LEGITIMATE-INTEREST`, `CONSENT`}
+             - if the set of Data Capture Fragments recommended for deletion is *the same as* the **Concerned Fragments** set, recommend status = `ACCEPT`,
+             - else, if the set of Data Capture Fragments recommended for deletion is *included in* the **Concerned Fragments** set, recommend status = `PARTIALLY-GRANTED`,
+             - else, recommend status = `DENIED`,
+             - when status is one of {`PARTIALLY-GRANTED`, `DENIED`} add one or more motives = `VALID-REASONS` or `IMPOSSIBLE` when **Concerned Fragments** that are not recommended for deletion, have in their `scope` Privacy Scope Triples included in the Eligible Privacy Scope under `CONTRACT` or `NECESSARY` Legal Bases, respectively.
+
+     - `MODIFY` Demands:
+         - If restricted to a Privacy Scope Restriction having a `processing-category` or a `purpose`, recommend status = `DENIED`, motive = `REQUEST-UNSUPPORTED` (A Delete request can only be relative to a category of data, not to a category of processing or a particular purpose.)
```

Comment:
Does this concern a DELETE or MODIFY request (or both)?
_A Delete request can only be relative to a category..._

Reply:
Both. I am updating the document